graph/graph.go

```diff
@@ -112,15 +113,17 @@ func NewTrafficMap() TrafficMap {
 }
 
 func Id(namespace, workload, app, version, service, graphType string) (id, nodeType string) {
-	// first, check for the special-case "unknown" node
-	if UnknownWorkload == workload && UnknownApp == app && "" == service {
-		return fmt.Sprintf("source-unknown"), NodeTypeUnknown
+	// first, check for the special-case "unknown" source node
+	if UnknownNamespace == namespace && UnknownWorkload == workload && UnknownApp == app && "" == service {
```

Comment:
Just curious - did you really mean to check to see if service is an empty string ("") or should this follow the pattern of everything else (UnknownService) ??

Reply:
The empty string check is correct.  It reflects that there is no "source" service.  The unknown node is always a source.